context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+The programming language is for people that want to be able to create digital art without learning how to render it in a different programming language. In ContextFree, the only code that the user needs to write involves the image that they produce. The user does not need to learn about graphics libraries or additional features of a more complex language. In general, ContextFree is optimal for someone who wants to create graphical images who has a basic understanding of computer science.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to produce simple shapes in the language because it includes `CIRCLE` and `SQUARE` instructions. In order to produce a simple unit circle, the user only needs to write one rule and does not need to specify any adjustments to the shape. Similarly, it is easy to color the background and size the canvas because of the supported methods in ContextFree. The language makes it intuitive to reposition elements with use of `x` and `y` transformations.
```

Comment:
I definitely agree with this sentiment. The simple shapes provided were definitely a huge time save, after seeing how difficult it was to do something like draw out letters. The way ContextFree was designed (with a series of rules) made repositioning elements as well as more complicated features like adjusting scaling factor, rotation factor, saturation, hue, among others, extremely intuitive.